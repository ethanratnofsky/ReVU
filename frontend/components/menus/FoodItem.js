import {View, ScrollView, Text, SafeAreaView} from 'react-native';
import menuPageStyles from './MenuStyles';

const FoodItem = ({item}) => {
    const diningHall = "Kissam Dining Hall";

    return (
        <View style={menuPageStyles.itemDisplay}>
            <Text style={menuPageStyles.itemText}>{item}</Text>
        </View>
    );

}

export default FoodItem;