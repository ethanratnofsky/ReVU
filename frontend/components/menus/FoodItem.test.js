import React from 'react';
import FoodItem from './FoodItem';
const renderer = require('react-test-renderer');

describe('<FoodItem />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<FoodItem />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});

it('renders correctly', () => {
    const tree = renderer.create(<FoodItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });