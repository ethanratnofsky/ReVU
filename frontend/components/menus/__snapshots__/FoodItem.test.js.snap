// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderColor": "#CFAE70",
      "borderWidth": 3,
      "height": 40,
      "justifyContent": "center",
      "marginBottom": 5,
      "marginLeft": 15,
      "marginRight": 15,
      "marginTop": 5,
      "padding": 5,
      "width": 300,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#1C1C1C",
        "fontFamily": "LibreCaslonText_400Regular",
      }
    }
  />
</View>
`;
